widgets/styles/templates/expanded-tiles/_index.scss

```diff
@@ -109,6 +109,7 @@ expanded-tiles:not(:empty) {
       border-radius: var(--expanded-tile-border-radius);
       grid-template-columns: minmax(0, dimens.$panel-left-width-lg) minmax(0, dimens.$panel-right-width-lg);
       grid-template-rows: 1fr;
+      min-height: 80vh;
```

Comment:
Shouldn't we follow the height specified in the Figma design?

Reply:
Also, now the swiper adapts to content height automatically

Reply:
Reverted. Thanks!